homeassistant/components/tahoma.py

```diff
@@ -0,0 +1,126 @@
+"""
+Support for Tahoma devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tahoma/
+"""
+from collections import defaultdict
+import logging
+import voluptuous as vol
+
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD, CONF_EXCLUDE
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import discovery
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import (slugify)
+
+REQUIREMENTS = ['tahoma-api==0.0.10']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'tahoma'
+
+TAHOMA_ID_LIST_SCHEMA = vol.Schema([cv.string])
```

Comment:
Add `cv.ensure_list`. You also don't need a separate schema for this. Just incorporate it in the `CONFIG_SCHEMA`.
```py
vol.Optional(CONF_EXCLUDE, default=[]): vol.All(cv.ensure_list, [cv.string]),
```